Sources/Sentry/SentryCrashIntegration.m

```diff
@@ -66,6 +67,7 @@ - (void)installWithOptions:(nonnull SentryOptions *)options
 {
     self.options = options;
     [self startCrashHandler];
+    sentry_install_async_hooks();
```

Comment:
`m`: I know I added this, but is it fine, that this might get called multiple times? If not should we add it to the `dispatchOnce` block below?

Reply:
I added an atomic inside the install function, that should avoid problems when calling multiple times.